docs/topics/templates/document-attributes.adoc

```diff
@@ -32,20 +46,19 @@
 // ****************
 // * Doc metadata *
 // ****************
-:DocInfoProductName: Migration Toolkit for Applications
-:DocInfoProductNumber: 5.2
-:DocInfoProductNameURL: migration_toolkit_for_applications
 
-:OpenShiftProductNumber: 4.9
-:ProductVersion: 5.2.1
+:OpenShiftProductNumber: 4.10
+:ProductVersion: 5.3.0
```

Comment:
I think the product version will be 1.0 for MTR here .
@PhilipCattanach , what do you think ?